addons/cookoff/initSettings.inc.sqf

```diff
@@ -67,3 +67,13 @@
     {[QGVAR(probabilityCoef), _this] call EFUNC(common,cbaSettings_settingChanged)},
     true // Needs mission restart
 ] call CBA_fnc_addSetting;
+
+[
+    QGVAR(finaleChance), "SLIDER",
+    [LSTRING(finaleChance_name), LSTRING(finaleChance_tooltip)],
+    LSTRING(category_displayName),
+    [0,1,0.5,2,true], // [min, max, default value, trailing decimals (-1 for whole numbers only)]
```

Comment:
```suggestion
    [0, 1, 0.5, 2, true], // [min, max, default value, trailing decimals (-1 for whole numbers only)]
```

Just a minor formatting thing.

Reply:
I fully agree with the change, but imo that can be done in #9758. Given the whole file is done in the current format, for consistency's sake I don't want to change it here.